pipeline/outputs/elasticsearch.md

```diff
@@ -8,54 +8,59 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
-| Write\_Operation | The write\_operation can be any of: create (default), index, update, upsert. | create |
-| Replace\_Dots | When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3. | Off |
-| Trace\_Output | Print all elasticsearch API request payloads to stdout \(for diag only\) | Off |
-| Trace\_Error | If elasticsearch return an error, print the elasticsearch API request and response \(for diag only\) | Off |
-| Current\_Time\_Index | Use current time for index generation instead of message record | Off |
-| Suppress\_Type\_Name | When enabled, mapping types is removed and `Type` option is ignored. If using Elasticsearch 8.0.0 or higher - it [no longer supports mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html), so it shall be set to On. | Off |
-| Workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `2` |
+| Key | Description | default | Overridable in NODE section of [Upstream](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) configuration |
+| :--- | :--- | :--- | :--- |
+| Host | IP address or hostname of the target Elasticsearch instance | 127.0.0.1 | Yes, default value is not applicable for NODE section of Upstream configuration, which **requires** host to be specified |
+| Port | TCP port of the target Elasticsearch instance | 9200 | Yes, default value is not applicable for NODE section of Upstream configuration, which **requires** port to be specified |
+| Path | Elasticsearch accepts new data on HTTP query path "/\_bulk". But it is also possible to serve Elasticsearch behind a reverse proxy on a subpath. This option defines such path on the fluent-bit side. It simply adds a path prefix in the indexing HTTP POST URI. | Empty string | Yes |
+| compress | Set payload compression mechanism. Option available is 'gzip' | | Yes |
+| Buffer\_Size | Specify the buffer size used to read the response from the Elasticsearch HTTP service. This option is useful for debugging purposes where is required to read full responses, note that response size grows depending of the number of records inserted. To set an _unlimited_ amount of memory set this value to **False**, otherwise the value must be according to the [Unit Size](../../administration/configuring-fluent-bit/unit-sizes.md) specification. | 4KB | Yes |
+| Pipeline | Newer versions of Elasticsearch allows to setup filters called pipelines. This option allows to define which pipeline the database should use. For performance reasons is strongly suggested to do parsing and filtering on Fluent Bit side, avoid pipelines. |  | Yes |
+| AWS\_Auth | Enable AWS Sigv4 Authentication for Amazon OpenSearch Service | Off | Yes |
+| AWS\_Region | Specify the AWS region for Amazon OpenSearch Service |  | Yes |
```

Comment:
```suggestion
The **Overridable** column indicates if a key can be overridden in the NODE section
of an
[Upstream](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md)
configuration.

| Key  | Description | Default | Overridable |
| :--- | :--- | :--- | :--- |
| Host | IP address or hostname of the target Elasticsearch instance. | `127.0.0.1` | Yes. Default value isn't applicable for NODE section of Upstream configuration, which **requires** Host to be specified. |
| Port | TCP port of the target Elasticsearch instance | `9200` | Yes. Default value isn't applicable for NODE section of Upstream configuration, which **requires** Port to be specified. |
| Path | Elasticsearch accepts new data on HTTP query path `/_bulk`. But it is also possible to serve Elasticsearch behind a reverse proxy on a subpath. This option defines such path on the fluent-bit side. It simply adds a path prefix in the indexing HTTP POST URI. | Empty string | Yes |
| compress | Set payload compression mechanism. Option available is `gzip`. | _none_ | Yes |
| Buffer\_Size | Specify the buffer size used to read the response from the Elasticsearch HTTP service. This option is useful for debugging purposes where is required to read full responses, note that response size grows depending of the number of records inserted. To set an _unlimited_ amount of memory set this value to **False**, otherwise the value must be according to the [Unit Size](../../administration/configuring-fluent-bit/unit-sizes.md) specification. | `4KB` | Yes |
| Pipeline | Newer versions of Elasticsearch allows to setup filters called pipelines. This option allows to define which pipeline the database should use. For performance reasons is strongly suggested to do parsing and filtering on Fluent Bit side, avoid pipelines. | _none_ | Yes |
| AWS\_Auth | Enable AWS Sigv4 Authentication for Amazon OpenSearch Service | `Off` | Yes |
| AWS\_Region | Specify the AWS region for Amazon OpenSearch Service | _none_ | Yes |
```

Reply:
Suggestion with modification - `Off` and `On` can be considered special values and not literals, so `Off` and `On` are not wrapped with backticks, `Empty string` is replaced with `_empty string_` for consistency with `_none_` - was applied in c620db5a2ca5a0cd0f5a6474e0db8a0139bca3c3, cdde1dec1e45673d9a9554429feea6df14f113c0 and 929d05ddfc37113904775d9e80e0d6f1fefcb1a9.